dFusion/dFusionSpec.md

```diff
@@ -131,13 +134,14 @@ Snark - applyAuction(
 	Private: balances
 	Private: orders
 	Private: touched balances + leaf number + balance merkle proofs per order,
-	Output: newstateRH
```

Comment:
The table VV - ordervolumes - need to be a private input

Reply:
That's volumeVector in line 133